scss/forms/_star-rating.scss

```
@@ -37,44 +36,85 @@
   }
 
   &:hover input + label {
-    @include form-star-rating($black);
+    @include form-star-rating($black, $black);
   }
 
   > input:hover ~ label {
-    @include form-star-rating($gray-500);
+    @include form-star-rating($gray-700, $white);
   }
 
   > input:hover + label {
-    @include form-star-rating($black);
+    @include form-star-rating($black, $black);
+  }
+
+  &.disabled,
+  &:disabled {
+    pointer-events: none;
+
+    > label {
+      @include form-star-rating($gray-700, $gray-700);
+    }
+
+    > input:checked ~ label {
+      @include form-star-rating($gray-500, $gray-500);
+    }
+
+    > input:checked + label {
+      @include form-star-rating($gray-700, $gray-700);
+    }
   }
 }
 
 .star-rating-dark {
   > label {
-    @include form-star-rating($brand-orange);
+    @include form-star-rating($brand-orange, $brand-orange);
   }
 
   > input:checked ~ label {
-    @include form-star-rating($gray-700);
+    @include form-star-rating($gray-500, $black);
```

```suggestion
    @include form-star-rating($gray-500, transparent);
```
```suggestion
    @include form-star-rating($gray-500, none);
```
I don't know if this should happen/work but might be better on gray backgrounds (if yes, spread it to the whole file with empty background stars)